<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>17</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>268</y>
      <width>115</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>Combobox Menu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Add Employee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Search Employee</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEmployee">
    <property name="title">
     <string>Employee Menu</string>
    </property>
    <addaction name="actionAdd_Employee"/>
    <addaction name="actionSearch_Employee"/>
   </widget>
   <addaction name="menuEmployee"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Employee">
   <property name="text">
    <string>Add Employee</string>
   </property>
  </action>
  <action name="actionSearch_Employee">
   <property name="text">
    <string>Search Employee</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
